### PR TITLE
Lds first session

### PR DESCRIPTION
**Pull request by eyalsingular**

### CODE REVIEW

File: ios/SingularBridge.m [hunk 1]
```diff
@@ -83,9 +83,27 @@ +(void)startSessionWithUserActivity:(NSUserActivity*)userActivity{
     };
     singularConfig.waitForTrackingAuthorizationWithTimeoutInterval =
         [[singularConfigDict objectForKey:@"waitForTrackingAuthorizationWithTimeoutInterval"] intValue];
-    
+
+    NSString* customUserId = [singularConfigDict objectForKey:@"customUserId"];
+
+    if (customUserId) {
```
Comment (tapasyat) [inline ios/SingularBridge.m:89]: are we allowing empty customUserId?

Comment (eyalsingular) [reply ios/SingularBridge.m:89]: yes

File: release-version.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+def update_docs():
+    print("Updating documentation")
+    platform = "react-native"
+    output, err = run_script("git describe --abbrev=0 --tags `git rev-list --tags --skip=1  --max-count=1`")
+    old_version = output.decode('utf-8').strip()
+    output, err = run_script("git describe --tags --abbrev=0")
+    new_version = output.decode('utf-8').strip()
+    os.chdir('sdk-tools/')
+    command = "./update_zendesk_articles.py --platform={0} --old-version={1} --new-version={2}".format(platform, old_version.replace('v', ''), new_version.replace('v', ''))
```
Comment (tapasyat) [inline release-version.py:51]: old_version/new_version.replace('v', '') - replace method not required.

**Review by tapasyat — APPROVED** [review]
looks great.

**Pull request merged**